src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js

```diff
@@ -10,16 +10,28 @@ export default class DocumentItemBody extends Component {
   }
 
   itemStatus = (item) => ({
-    IsOnShelf: () =>
-      invenioConfig.ITEMS.canCirculateStatuses.includes(item.status) &&
-      !item.circulation,
+    isOnCirculation: () =>
+      invenioConfig.ITEMS.canCirculateStatuses.includes(item.status),
+    isOnShelf: () => !item.circulation,
   });
 
   statusLabel = (item) => {
-    const onShelf = this.itemStatus(item).IsOnShelf();
+    const circulationStates = {
+      ITEM_AT_DESK: 'At Desk',
+      ITEM_ON_LOAN: 'On Loan',
+      ITEM_IN_TRANSIT_FOR_PICKUP: 'In Transit For Pickup',
+      ITEM_IN_TRANSIT_TO_HOUSE: 'In Transit To House',
+    };
```

Comment:
For maintainability it is better to import objects from central files, i searched around the project and in the config/defaultConfig.js there is a records_config file with a property called "loanActiveStates" that looks pretty similar 😜

Reply:
You are right, still, I cannot just use the defaultConfig, as I would need each element of the loanActiveStates to have a key (ej: ITEM_ON_LOAN) and a value (ej: 'On Loan'),  currently I have to manually add the value so I cannot take a lot of advantage of the config.

One possibility would be to had this object (circulationStates) to the config so it's easily reusable, but I don't even know if this is the solution that we should adopt (we could go for something simpler, like just putting 'On Loan' for everything).

Reply:
To make it simple, no need for the various circulationStates above, please just change it to:
```
if (canCirculate) {
     return onShelf ? (
        <span className="success">On shelf</span>
      ) : (
        <span>On loan</span>
      );
```
Should we give it a color when on loan ? orange maybe? ping @kprzerwa

Reply:
Had a small chat with @kprzerwa and @zzacharo. The final conclusion is to keep it grey.